components/src/core/components/ListTable/ListTable.vue

```diff
@@ -4,13 +4,13 @@
       <oxd-card-tr :clickable="false">
         <oxd-card-th
           v-if="loading && skeleton"
-          class="oxd-padding-cell oxd-table-th"
+          class="oxd-padding-cell oxd-table-th checkbox-cell"
         >
           <oxd-skeleton class="oxd-skeleton-checkbox" animateHighlighted />
         </oxd-card-th>
         <oxd-card-th
           v-else-if="selectable"
-          class="oxd-padding-cell oxd-table-th"
+          class="oxd-padding-cell oxd-table-th checkbox-cell"
```

Comment:
this is the header row. do you need it affect there also?

Reply:
Yes, it should change as well.